test/output_test.h

```diff
@@ -201,6 +204,40 @@ namespace {
 
 const char* const dec_re = "[0-9]*[.]?[0-9]+([eE][-+][0-9]+)?";
 
+inline int SubstrCnt(const std::string& haystack, const std::string& pat) {
```

Comment:
personally, i'd declare it here and implement it in a source file. but that's not essential.

Reply:
Why not.